src/booleans.js

```diff
@@ -1,61 +1,88 @@
 const negate = (a) => {
-  // your code here
+  return 1 > a
 };
 
 const both = (a, b) => {
-  // your code here
+  return a > 0 && b > 0
 };
 
 const either = (a, b) => {
-  // your code here
+  return a > 0 || b > 0
 };
 
 const none = (a, b) => {
-  // your code here
+  if (a || b) {
+    return false
+  } else {
+    return true
+  }
 };
 
 const one = (a, b) => {
-  // your code here
+  if ((a !== b) && (a || b)) {
+    return true 
+  } else {
+    return false
+  }
 };
 
 const truthiness = (a) => {
-  // your code here
+  if (a) {
+    return true
+  } else {
+    return false
+  }
 };
 
 const isEqual = (a, b) => {
-  // your code here
+  return a === b
 };
 
 const isGreaterThan = (a, b) => {
-  // your code here
+  return a > b
 };
 
 const isLessThanOrEqualTo = (a, b) => {
-  // your code here
+  return a <= b
 };
 
 const isOdd = (a) => {
-  // your code here
+  if (a % 2 === 0) {
+    return false
+  } else { return true}
```

Comment:
You can also return the expression directly here, prepended with the not operator `!` to reverse the truthiness.

If you ever do need to use `if` and `else`, ensure you maintain consistency with line breaks:

```js
if (1 === 1) {
  ...
} else {
  ...
}
```